app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -1,3 +1,5 @@
+require 'hawkular/hawkular_client_utils'
```

Comment:
why do you require this? I dont see any new methods that could from from there in the patch for this file.